test/helpers/kubectl.go

```diff
@@ -508,7 +508,6 @@ func (kub *Kubectl) ParallelResourceDelete(namespace, resource string, names []s
 			if !res.WasSuccessful() {
 				ginkgoext.By("Unable to delete %s %s with '%s': %s",
 					resource, name, cmd, res.OutputPrettyPrint())
-
```

Comment:
Most of the changes in this file appear to be unrelated, and should be reverted.

Reply:
It seems I unknowingly had `gofumpt` enabled :sweat_smile: